[CUBRIDMAN-222] DBLink설명란에 단락 구분 및 althost관련 예제 추가 (#535)
CUBRID/cubrid-manual: en/sql/tuning.rst

    SET OPTIMIZATION LEVEL opt-level [;]
    GET OPTIMIZATION LEVEL [ { TO | INTO } variable ] [;]

> *   *opt-level* : This value specifies the optimisation level and has the following meanings, and any other value will result in an error. 

Reviewer: ```suggestion
*   *opt-level* : This value specifies the optimization level and is associated with the following settings. Any other value will result in an error. 
```

Author: 오타까지 ... 상세한 리뷰 감사합니다. 
수정/반영하였습니다.
